### PR TITLE
Update in relation to Ionic CLI release 6.9.0

### PR DESCRIPTION
**Pull request by Heerschop**
[Change log Ionic CLI release 6.9.0](https://github.com/ionic-team/ionic-cli/blob/develop/packages/%40ionic/cli/CHANGELOG.md#690-2020-05-11)

### CODE REVIEW

**Comment by dwieeb** [comment]
Hi @Heerschop, thanks for adding documentation to the hooks you made! 😁 

I know it's probably pretty obvious, but can we explicitly say that these hooks won't run for the `ionic cordova` commands?

**Comment by Heerschop** [comment]
Update the text as suggested :-)

File: src/pages/cli/configuration.md [hunk 1]
```diff
@@ -67,6 +67,23 @@ The CLI can run scripts during certain events, such as before and after builds.
 * `ionic:serve:after`: executed after the dev server is terminated
 * `ionic:build:before`: executed before a web asset build begins
 * `ionic:build:after`: executed after a web asset build finishes
+* `ionic:capacitor:run:before` : executed on capacitor run before capacitor open is executed
+* `ionic:capacitor:build:before` : executed on capacitor build before capacitor open is executed
+
+The capacitor hooks are executed after the `ionic:serve` and `ionic:build` hooks, they will only be executed when using the `ionic capacitor build` or the `ionic capacitor run` command. When using a shell script for any of the hooks, several environment variables are set containing context information.
```
Comment (dwieeb) [inline src/pages/cli/configuration.md:73]:
```suggestion
The Capacitor hooks are executed after the `ionic:serve` and `ionic:build` hooks. They will only be executed when using the `ionic capacitor build` or `ionic capacitor run` commands. When using a shell script for any of the hooks, several environment variables are set containing context information.
```

File: src/pages/cli/configuration.md [hunk 2]
```diff
@@ -67,6 +67,23 @@ The CLI can run scripts during certain events, such as before and after builds.
...
+
+The following example, shows environment variables set for the `ionic:capacitor:build:before` hook.
+```shell
```
Comment (dwieeb) [inline src/pages/cli/configuration.md:76]:
```suggestion
The following example shows the environment variables that are set for the `ionic:capacitor:run:before` and `ionic:capacitor:build:before` hooks.

```shell
```

**Comment by dwieeb** [comment]
Just a few nitpicky sentence structure changes, then this will be ready to go!

**Pull request merged**